qchem/CHANGELOG.md

```diff
@@ -4,13 +4,6 @@
 
 <h3>Improvements</h3>
 
-* The ``meanfield`` function has been modified to avoid creating
```

Comment:
This entry was merged into the release candidate branch too and is included in the v0.19.0 release (#1858). Therefore, the v0.20.0 changelog sections were reset.